Core/FileLoaders/LocalFileLoader.cpp

```diff
@@ -126,6 +126,10 @@ size_t LocalFileLoader::ReadAt(s64 absolutePos, size_t bytes, size_t count, void
 		lseek64(fd_, absolutePos, SEEK_SET);
 		return read(fd_, data, bytes * count) / bytes;
 	}
+#elif defined(__wiiu__)
+	std::lock_guard<std::mutex> guard(readLock_);
+	lseek(fd_, absolutePos, SEEK_SET);
```

Comment:
Is this 64-bit clean?  Some ISOs (hacks, translations) are larger than 2GB and have had issues with lseek/etc.

-[Unknown]

Reply:
lseek uses off_t which is 64-bit here. 
I did test a 2.5gb iso a bit but not long enough to know for sure if things are okay beyond the 2GB boundary.
lseek64 was not defined in the headers so I couldn't use that.
the special case here is having 64bit file offsets but no pread (I had a wrapper before but that wasn't thread safe), which is why I added the ifdef.

Reply:
Ah, okay, cool.

-[Unknown]